prisma/schema.prisma

```diff
@@ -49,15 +51,18 @@ model Session {
 }
 
 model User {
-    id            String     @id @default(cuid())
-    name          String?
-    email         String?    @unique
-    emailVerified DateTime?
-    image         String?
-    role          Role       @default(READER)
-    accounts      Account[]
-    sessions      Session[]
-    BlogPost      BlogPost[]
+    id              String     @id @default(cuid())
+    name            String?
+    email           String?    @unique
+    emailVerified   DateTime?
+    image           String?
+    role            Role       @default(READER)
+    accounts        Account[]
+    sessions        Session[]
+    BlogPost        BlogPost[]
+    Comment         Comment[]
```

Comment:
Can we rename the 'BlogPost' and 'Comment' to 'blogPosts' and 'comments' so that it matches the convention?

Also the usage would make more sense 'user.blogPosts' and 'user.comments' over 'user.BlogPost' and 'user.Comment'